packages/commonjs/src/helpers.js

```diff
@@ -70,12 +49,27 @@ export function getAugmentedNamespace(n) {
 `;
 
 const HELPER_NON_DYNAMIC = `
+export function createCommonjsModule(fn) {
+  var module = { exports: {} }
+	return fn(module, module.exports), module.exports;
+}
+
 export function commonjsRequire () {
 	throw new Error('Dynamic requires are not currently supported by @rollup/plugin-commonjs');
```

Comment:
If you are already on this one... Please change that damn message :-)
Should be something like `"Please configure `dynamicRequireTargets` appropriately for this `require` call to behave properly (see the docs)"`

Reply:
That is a really good idea, I love error messages that educate the user. I change it so the handler is now

```js
function commonjsRequire (target) {
  throw new Error('Could not dynamically require "' + target + '". Please configure the ' + 
    'dynamicRequireTargets option of @rollup/plugin-commonjs appropriately for this require ' +
    'call to behave properly.');
}
```

This should give the user also some context where to look, even though it cannot tell you which module was the importer.